graviti/client/branch.py

```diff
@@ -50,18 +64,88 @@ def list_branches(  # pylint: disable=unused-argument
         offset: The offset of the page.
         limit: The limit of the page.
 
-    Return:
+    Returns:
         The response of OpenAPI.
 
+    Examples:
+        List branches:
```

Comment:
shall we use *italic* for example titles?

Reply:
yes, but "List branches" and "Get the branch with the given name" here are descriptions.